src/app/src/components/AboutSlide.js

```diff
@@ -48,6 +48,9 @@ export default class AboutSlide extends Component {
             this.autoplay();
         }
     }
+    setVideoRef(ref) {
+        this.videoRef = ref;
+    }
```

Comment:
What is the purpose of this function? It doesn't seem to be called anywhere...

Reply:
Good catch, remnant of something I was trying out.